Setup/InstallData.php

```diff
@@ -70,29 +44,5 @@ public function install(
         catch (\Magento\Framework\Exception\LocalizedException $ex) {
             $this->_state->setAreaCode(\Magento\Framework\App\Area::AREA_ADMINHTML);
         }
```

Comment:
Looks like this file doesn't do anything anymore. So it could be safely removed. Right?